src/content/guides/using-persistent-volumes-on-aws-with-efs/index.md

```diff
@@ -0,0 +1,126 @@
+- EFS has the capability to mount the same Persistent Volume to multiple pods at the same time using the ReadWriteMany [access mode](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes).
+- EFS will not hit the [AWS Instance Volume Limit](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/volume_limits.html) as it is a software mount and will avoid the [Impaired EBS](https://docs.giantswarm.io/guides/aws-impaired-volumes/) issue.
+- EFS mount times are better than EBS.
+
+If you need to use EFS to provision volumes, be advised:
+- All Kubernetes Persistent Volumes will be stored in the same EFS instance. You can deploy multiple provisioners per cluster, each having its own storage-class and EFS instance.
+- [EFS throughtput](https://docs.aws.amazon.com/efs/latest/ug/performance.html) need to be set up accordingly in order to not have performance issues. We only recommend Provisioned Throughput, and if you need high performance you will need EBS.
+- EFS backups are done with [AWS Backup](https://aws.amazon.com/backup/) and it does not have the snapshot feature of EBS.
+- You cannot limit the amount of data stored in an EFS volume. The requested value in Kubernetes is ignored.
+- EFS mount targets are limited to 1 subnet per Availability Zone. Each NodePool will create a different subnet per AZ, plan accordingly. 
+
+
+## Provision an EFS instance on AWS
+
+- Open the [AWS management](https://aws.amazon.com/console/) console in the account your cluster is located.
+- Select EFS from the services list.
+- Create a new EFS mount and select the VPC where your cluster is located.
+- Select the Availability Zone with the subnets your instances are located on and the security-groups of the workers.
+- Choose the throughput and performance mode, no file system policy or access points are needed.
+- Create the instance and note the EFS instance id.
+
+## EFS Provisioner configuration file
+Create a file on your local machine named "efs-provisioner.yaml" with the following content:
+```yaml
+global:
+  deployEnv: production
+
+efsProvisioner:
+  efsFileSystemId: fs-90f935c8
+  awsRegion: eu-central-1
+  path: /
+  provisionerName: giantswarm.io/aws-efs
+  storageClass:
+    name: efs
+    isDefault: false
+    gidAllocate:
+      enabled: true
+      gidMin: 40000
+      gidMax: 50000
+    reclaimPolicy: Delete
+    mountOptions: []
+
+rbac:
+  create: true
+
+podSecurityPolicy:
+  enabled: true
+```
+You will need to populate the **efsFileSystemId** and **awsRegion** parameters to match the configured values of the EFS instance.
+
+For additional configuration parameters see [documentation](https://github.com/kubernetes-incubator/external-storage/tree/master/aws/efs).
+
+## Installing EFS Provisioner
+
+To install the provisioner you will need to follow these steps:
+- Access Giant Swarm web UI and select the cluster on which you want to install the provisioner.
+- Open the "Helm Stable"
```

Comment:
Sorry for deleting the word catalog after "Helm Stable" but I couldn't find a way to comment while editing.
That being said, there is no longer a "Helm Stable" catalog in the company jargon. It is either the "Giant Swarm App Catalog" or the "Playground App Catalog"

Reply:
No problem! If we are going to drop that Catalog then we might need to maintain this ourselves... which I am not sure we want to do for now.

I would like to keep that catalog if possible as there are some applications that work out of the box from upstream but I understand it can be confusing for customers